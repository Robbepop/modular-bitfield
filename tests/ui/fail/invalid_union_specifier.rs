use modular_bitfield::prelude::*;

#[derive(BitfieldSpecifier)]
pub union InvalidUnionSpecifier {
    a: bool,
    b: B7,
    c: u8,
}

fn main() {}
