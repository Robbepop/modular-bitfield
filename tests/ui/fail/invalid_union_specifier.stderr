error: unions are not supported as bitfield specifiers
 --> tests/ui/fail/invalid_union_specifier.rs:4:1
  |
4 | pub union InvalidUnionSpecifier {
  | ^^^
